src/en/proposals/changelings/veritus.md

```diff
@@ -0,0 +1,33 @@
+# Changelings [Veritius, Unapproved]
+
+Changelings should be genuinely terrifying. Like, serious The Thing shit. A changeling should make the crew collectively shit its pants.
+
+## Mechanics
+### Biomass
+Drastic modifications (such as extruding chitin armor) should require biomass. Changelings obviously can't pull matter out of thin air (that's the job of a wizard), so it needs to get it from somewhere, right? And guess what's a great source of biomass - the crew. (You shouldn't be able to produce it though, you gotta eat people.)
+
+### Undetectable
+A changeling in human form should be completely undetectable by reliable means. No technology can detect a changeling when it's trying to hide.
+
```

Comment:
why not have the thing where you can put their blood in a beaker, and then heat it up to detect them?